pageserver/src/tenant/storage_layer/layer_desc.rs

```diff
@@ -19,14 +19,21 @@ use serde::{Deserialize, Serialize};
 pub struct PersistentLayerDesc {
     pub tenant_id: TenantId,
     pub timeline_id: TimelineId,
+    /// Range of keys that this layer covers
     pub key_range: Range<Key>,
-    /// For image layer, this is `[lsn, lsn+1)`.
+    /// Inclusive start bound of the LSN range that this layer holds
+    /// Exclusive end bound of the LSN range that this layer holds.
+    ///
+    /// - For an open in-memory layer, this is MAX_LSN.
```

Comment:
What is `this` in this context? The start or the end bound?

Reply:
I don't think the start_lsn can be MAX_LSN, so that leaves `lsn_range.end`, which I think was in place in code as well, yes: https://github.com/neondatabase/neon/blob/0b001a00013df8684d65f2b32afd42ce7fd01ae9/pageserver/src/tenant/storage_layer/inmemory_layer.rs#L106-L108

Reply:
31276dfe4ac2e8dc07a765a31d709b5ca041cadf